app/src/main/java/com/epfl/drawyourpath/database/Database.kt

```diff
@@ -2,58 +2,43 @@ package com.epfl.drawyourpath.database
 
 import android.graphics.Bitmap
 import com.epfl.drawyourpath.path.Run
-import com.epfl.drawyourpath.userProfile.UserModel
 import com.epfl.drawyourpath.userProfile.dailygoal.DailyGoal
 import java.util.concurrent.CompletableFuture
 
-abstract class Database {
-    companion object {
-        // name of the different attributes inside the user account
-        const val usernameFile: String = "username"
-        const val firstnameFile: String = "firstname"
-        const val surnameFile: String = "surname"
-        const val emailFile: String = "email"
-        const val dateOfBirthFile: String = "dateOfBirth"
-        const val profilePhotoFile: String = "profilePhoto"
-        const val friendsListFile: String = "friendsList"
-
-        // current daily goal
-        const val currentDistanceGoalFile: String = "distanceGoal"
-        const val currentActivityTimeGoalFile: String = "activityTimeGoal"
-        const val currentNOfPathsGoalFile: String = "nbOfPathsGoal"
-
-        // daily user goals
-        const val dailyGoalsFile: String = "dailyGoals"
-        const val expectedDistanceFile: String = "expectedDistance"
-        const val expectedActivityTimeFile: String = "expectedActivityTime"
-        const val expectedNbOfPathsFile: String = "expectedNbOfPaths"
-        const val obtainedDistanceFile: String = "obtainedDistance"
-        const val obtainedActivityTimeFile: String = "obtainedActivityTime"
-        const val obtainedNbOfPathsFile: String = "obtainedNbOfPaths"
-
-        // user achievements, will be used later for the trophies
-        const val achievementsFile: String = "achievements"
-        const val totalDistanceFile: String = "totalDistance"
-        const val totalActivityTimeFile: String = "totalActivityTime"
-        const val totalNbOfPathsFile: String = "totalNbOfPaths"
-
-        // run history
-        const val runsHistoryFile: String = "runsHistory"
-    }
+data class UserGoals(
+    val paths: Long? = null,
+    val distance: Double? = null,
+    val activityTime: Long? = null,
+)
+
+data class UserData(
+    val username: String? = null,
+    val userId: String? = null,
+    val firstname: String? = null,
+    val surname: String? = null,
+    val birthDate: Long? = null,
+    val email: String? = null,
+    val goals: UserGoals? = null,
+    val picture: String? = null,
+    val friendList: List<String>? = null,
+    val runs: List<Run>? = null,
+    val dailyGoals: List<DailyGoal>? = null,
+)
```

Comment:
Shouldn't it be done in the achievement PR?

Reply:
I think it is better to keep them since there are already used by @HoLuDyEr and will be used in the next task like the trophies or at the end of a path drawing .